src/nodes.ts

```diff
@@ -157,9 +168,11 @@ export class AttachedServersNode implements INode {
 		const index = this._attachedServers.findIndex((value) => value.id === server.id);
 		if (index !== -1) {
 			const deletedNodes = this._attachedServers.splice(index, 1);
-			await this._azureAccount.credentials.deleteSecret(this._serviceName, server.id);
-			await this.persistIds();
-			return deletedNodes;
+			if (this._keytar) {
+				await this._keytar.deletePassword(this._serviceName, server.id);
+				await this.persistIds();
+				return deletedNodes;
```

Comment:
I don't have much context here, but maybe the last two lines should be run even when keytar could not be loaded? (similar above)